diff -urN a/Makefile b/Makefile
--- a/Makefile	2019-05-03 12:45:43.000000000 -0500
+++ b/Makefile	2022-01-06 11:42:35.000000000 -0600
@@ -14,7 +14,7 @@
 CC    := $(CROSS_COMPILE)gcc
 STRIP := $(CROSS_COMPILE)strip
 
-CFLAGS += -O2 -Wall -Wextra -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wpointer-arith -Wno-unused-parameter
+CFLAGS += -Wall -Wextra -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wpointer-arith -Wno-unused-parameter
 LDFLAGS += -lpthread
 SHARED = -shared -Wl,-soname,libhdhomerun$(LIBEXT)
 
@@ -32,9 +32,8 @@
     LDFLAGS += -lsocket
   endif
   ifeq ($(OS),Darwin)
-    CFLAGS += -arch x86_64
     LIBEXT := .dylib
-    SHARED := -dynamiclib -install_name libhdhomerun$(LIBEXT)
+    SHARED := -dynamiclib -install_name @PREFIX@/lib/libhdhomerun$(LIBEXT)
   endif
 endif
 
